### PR TITLE
Make chip and tip public + more tests

### PR DESCRIPTION
**Pull request by gbalabasquer**

### CODE REVIEW

**Review by brianmcmichael — APPROVED** [review]
++ these definitely need to be public.

Tests look good, too.

File: src/test/clip.t.sol [hunk 1]
```diff
@@ -1138,6 +1173,65 @@ contract ClipperTest is DSTest {
...
+        clip.redo(1, address(345));
+        assertEq(vat.dai(address(345)), clip.chip() * tab / WAD);
+
+        vat.file(ilk, "dust", rad(110 ether) + 1); // 1 wei > than $110 (tab) as dust
+
+        hevm.warp(now + 300);
+        clip.redo(1, address(456));
+        assertEq(vat.dai(address(456)), 0);
+
+        vat.file(ilk, "dust", rad(20 ether)); // $20 dust
```
Comment (e18r) [inline src/test/clip.t.sol:1208]: add comment here explaining that you are testing the dusty lot case

**Review by e18r — APPROVED** [review]
LGTM

File: src/test/clip.t.sol (hunk 1 above)
Comment (gbalabasquer) [reply src/test/clip.t.sol:1206]: done!

File: src/test/clip.t.sol [hunk 2]
```diff
@@ -1138,6 +1173,65 @@ contract ClipperTest is DSTest {
         assertTrue(try_redo(1, address(this)));
     }
 
+    function test_redo_incentive() public takeSetup {
+        // (, uint256 price) = clip.getStatus(1);
+        // assertEq(price, ray(5 ether));
```
[the remark this answers is not in the document]
Comment (gbalabasquer) [reply src/test/clip.t.sol]: Done!

**Pull request merged**